assets/src/less/store-lists.less

```
@@ -248,6 +248,16 @@
 
 #dokan-seller-listing-wrap {
 
+    // The listing is laid out in percentages that already account for their own
```

Stylelint (17.14.1)

[error] Unexpected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)